packages/teams-js/src/public/stageView.ts

```diff
@@ -115,4 +115,39 @@ export namespace stageView {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
   }
+
+  export namespace stageClose {
```

Comment:
This namespace should also have the `@beta` tag added to it's comment, along with a brief comment explaining what the purpose of the capability is. Those comments and tags will get turned into dev documentation automatically.